service/src/test/java/bio/terra/pipelines/common/utils/StairwaySetPipelineRunStatusHookTest.java

```diff
@@ -7,6 +7,7 @@
 
 import bio.terra.pipelines.db.entities.PipelineRun;
 import bio.terra.pipelines.db.repositories.PipelineRunsRepository;
+import bio.terra.pipelines.db.repositories.PipelinesRepository;
```

Comment:
was this just missing by accident before?

Reply:
huh i have no idea why that was added, i removed it and tests still seem to be passing.  I prob messed up something while i was refactoring stuff 🤷